README.md

```diff
@@ -21,13 +21,13 @@ To setup on your local machine:
 3. Run the generate conda file script and create a conda environment:   
     ```
     cd Recommenders
-    ./scripts/generate_conda_file.sh
-    conda env create -n reco -f conda_bare.yaml  
+    python scripts/generate_conda_file.py
+    conda env create -f reco_base.yaml  
```

Comment:
Maybe mention about GPU env too, here or above line referring to SETUP.md

Reply:
good idea, added reference to setup.md